lib/util.js

```diff
@@ -0,0 +1,25 @@
+var url = require("url");
+var qs = require("qs");
+
+/**
+ * getNextLink
+ * @function
+ * @param {Object} response A response object returned from calling 'client.makeRequest'
+ * @returns A parsed version of the link to the subsequent page, or null if no such page exists.
+ */
+var getNextLink = function (response) {
```

Comment:
I've re-implemented this functionality using querystring instead of qs, which is a core node library: https://nodejs.org/api/querystring.html